server/command.go

```diff
@@ -141,6 +161,10 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 				return &model.CommandResponse{}, nil
 			} else if len(optionList) == 1 {
 				features = optionList[0]
+				if !areFeaturesValid(features) {
```

Comment:
1/5 I would prefer this function call to return an array of bad features and print them out so the user.  However, given there are currently only a max of 7 specified features, maybe that is overkill.  I will defer to @marianunez to see if she thinks it's worth the change.

Otherwise, this methodology works for me!

Reply:
Makes sense. I'm up for adding it. Waiting for @marianunez 's go-ahead.

Reply:
Makes sense. I'm up for adding it if needed.